### DISCUSSION TITLE
Docker?

### DISCUSSION THREAD

**Discussion by ArchonMegalon** [Xerophayze/XeroFlow#5] [General]
Basically the titel. 
This really looks interesting and it runs on linux already (or at least it looks like it before testing that because of the setup.sh and everything being py). 

Why not distribute it as docker image (and additionaly as docker compose  - and even better - portainer template)?

I really dislike installing (not) self written apps directly on my host, but with docker that is a none-issue...

**Comment by Xerophayze** [comment]
Setting up a docker I think will be on the list. I'm not super familiar
with how to do that, but I don't think it'll be too hard to figure out.
